test/integration/image-optimizer/test/minimum-cache-ttl.test.ts

```diff
@@ -22,5 +22,6 @@ describe('with minimumCacheTTL of 5 sec', () => {
     },
     appDir,
     imagesDir,
+    isrFlushToDisk: false,
```

Comment:
```suggestion
    isrFlushToDisk: true,
```

This one seems important to flush to disk so we run the corresponding tests